core/src/main/resources/jenkins/triggers/ReverseBuildTrigger/config_lt.properties

```diff
@@ -0,0 +1,4 @@
+Projects\ to\ watch=Steb?ti projektus
```

Comment:
This looks like an encoding issue.

Or Lithuanian is weirder than my imagination can handle 😉